app/controllers/providers/confirm_dwp_non_passported_applications_controller.rb

```diff
@@ -70,7 +70,9 @@ def correct_dwp_result?
     end
 
     def hmrc_call_enabled?
-      Rails.configuration.x.collect_hmrc_data
+      return true if Rails.configuration.x.collect_hmrc_data.eql?(true)
```

Comment:
Can this be removed now?

Reply:
Yep. Good spot. I'll make this change now